src/announcement/announcement.test.ts

```diff
@@ -0,0 +1,54 @@
+//eslint-disable-next-line
+require("dotenv").config();
+import Web3 from "web3";
+import { keccak256 } from "js-sha3";
+
+import { batch } from "./announcement";
+
+const TESTING_PRIVATE_KEY = process.env.TESTING_PRIVATE_KEY as string;
+const RPC_URL = process.env.RPC_URL as string;
```

Comment:
Curious if `String(process.env.TESTING_PRIVATE_KEY)` is better than `as string`...